src/functionality.js

```diff
@@ -85,12 +87,33 @@ body.addEventListener('click', (e) => {
       localStorage.setItem('taskArray', JSON.stringify(x));
     });
   }
+  // When the user clicks on the checkbox the obj updates to checked
+  if (e.target.classList.contains('checkbox')) {
+    const tasks = JSON.parse(localStorage.getItem('taskArray'));
+    updateTaskArray(e.target, tasks);
+  }
+});
+
```

Comment:
- [optional]Everything works fine. Great job. please, the round checkbox makes it difficult to click as you must take time to centralize the mouse pointer. If possible, you put a square checkbox. Nice work